src/status_im/navigation/screens.cljs

```diff
@@ -631,6 +632,10 @@
      :options   options/transparent-modal-screen-options
      :component settings.messages/view}
 
+    {:name      :screen/syncing
```

Comment:
i don't know if we have any rule for this but probably `:screen.settings/syncing` would be better

Reply:
Thank you @flexsurfer for the review.

I am not sure but we currently can't use that. These screen qualifications are getting removed while navigating by `clj->js` and we are then manually adding `:screen` to all new screens.

more info - https://github.com/status-im/status-mobile/pull/18778#discussion_r1490507009

cc @J-Son89

Reply:
But it could be screen/settings.syncing which is fine, no?